src/java/fr/paris/lutece/plugins/example/modules/elastic/business/Project.java

```diff
@@ -0,0 +1,109 @@
+ * modification, are permitted provided that the following conditions
+ * are met:
+ *
+ *  1. Redistributions of source code must retain the above copyright notice
+ *     and the following disclaimer.
+ *
+ *  2. Redistributions in binary form must reproduce the above copyright notice
+ *     and the following disclaimer in the documentation and/or other materials
+ *     provided with the distribution.
+ *
+ *  3. Neither the name of 'Mairie de Paris' nor 'Lutece' nor the names of its
+ *     contributors may be used to endorse or promote products derived from
+ *     this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDERS OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+ * License 1.0
+ */ 
+package fr.paris.lutece.plugins.example.modules.elastic.business;
+
+import javax.validation.constraints.*;
+import org.hibernate.validator.constraints.*;
+import java.io.Serializable;
+
+/**
+ * This is the business class for the object Project
+ */ 
+public class Project implements Serializable
+{
+    private static final long serialVersionUID = 1L;
+
+    // Variables declarations 
+    private int _nId;
+    
+    @NotEmpty( message = "#i18n{module.example.elastic.validation.project.Name.notEmpty}" )
+    @Size( max = 50 , message = "#i18n{module.example.elastic.validation.project.Name.size}" ) 
+    private String _strName;
+    
+    private int _nCout;
+
+    /**
+     * Returns the Id
+     * @return The Id
+     */
+    public int getId( )
+    {
+        return _nId;
+    }
+
+    /**
+     * Sets the Id
+     * @param nId The Id
+     */ 
+    public void setId( int nId )
+    {
+        _nId = nId;
+    }
+    
+    /**
+     * Returns the Name
+     * @return The Name
+     */
+    public String getName( )
+    {
+        return _strName;
+    }
+
+    /**
+     * Sets the Name
+     * @param strName The Name
+     */ 
+    public void setName( String strName )
+    {
+        _strName = strName;
+    }
+    
+    /**
+     * Returns the Cout
+     * @return The Cout
+     */
+    public int getCout( )
+    {
+        return _nCout;
+    }
+
+    /**
+     * Sets the Cout
+     * @param nCout The Cout
+     */ 
+    public void setCout( int nCout )
+    {
+        _nCout = nCout;
+    }
+}
```

Comment:
Why is this business class needed in this project ? How is it different from the Project class in the plugin-example project